web/src/js/components/ProxyApp.jsx

```diff
@@ -11,11 +11,11 @@ import Modal from './Modal/Modal'
 
 class ProxyAppMain extends Component {
 
-    componentWillMount() {
+    componentDidMount() {
         window.addEventListener('keydown', this.props.onKeyDown);
     }
 
-    componentWillUnmount() {
+    UNSAFE_componentWillUnmount() {
```

Comment:
Unmount is not prefixed with `UNSAFE_`, is it?

https://reactjs.org/docs/react-component.html#componentwillunmount

Reply:
I made a mistake. I'll fix it.